docs/domains/Domain.json

```diff
@@ -183,6 +183,14 @@
           "description": "Model in image model files and properties.",
           "$ref": "#/definitions/Model"
         },
+        "distributionStrategy": {
+          "description": "The strategy for handling changes to the secrets or overridesConfigMap fields. Legal values are DYNAMIC and ON_RESTART.",
+          "type": "string",
+          "enum": [
```

Comment:
All descriptions for 'introspectVersion' should also have something like 'See also 'distributionStrategy'.'.